doc/cheatsheet.md

```diff
@@ -0,0 +1,255 @@
+% djot cheatsheet
+
+## Basics
+
+| Markup | Result |
+| --- | --- |
+| `_italic_` or `{_italic_}` | *italic* |
+| `*bold*` or `{*bold*}` | **bold** |
+| `` `verbatim/code` `` | `verbatim/code` |
+| `H~2~O` | H<sub>2</sub>O |
+| `20^th^` | 20<sup>th</sup> |
+| `{=highlighted=}` | <mark>highlighted</mark> |
+| `{+insert+}` | <ins>insert</ins> |
+| `{-delete-}` | <del>delete</del> |
+
+Use triple-backticks for fenced verbatim blocks:
+
+~~~~~~
+```
+func say-hello(nm) {
+    print("hello ${nm}!");
+}
+```
+~~~~~~
+
+**Block quote**:
+
+~~~
+> Some blockquoted
+> content in here.
+~~~
+
+**Comments** `{% look like this`\
+`and can span multiple lines %}`.
+
+**Emojis**: `:+1:` → 👍, `:smiley:` → 😃 , etc.
+
+**Thematic break** (horizontal line): `***` or `---` on its own line.
+Can be longer than three characters, and may contain or be indented
+by spaces/tabs.
+
+
+
+## Math
+
+Use LaTeX formatting. `` $`p = mv` `` for inline math, and
+
+`` $$`E = K + U` ``
+
+for display math.
+
+
+## Special Characters
+
+  * `...` → … (ellipsis)
+  * `--` → – (an n-dash)
+  * `---` → — (an m-dash)
```

Comment:
I think the technical terms are en-dash and em-dash

Reply:
Should we have something about straight quotes becoming curly?

Reply:
Thanks.